src/main/java/Statistics.java

```diff
@@ -0,0 +1,57 @@
+public class Statistics {
+    public static double avgSalary(Employee[] employees) {
+        if (employees != null) {
+            return totalSalarySum(employees) / employees.length;
+        } else {
+            return 0;
+        }
+    }
+
+    private static double totalSalarySum(Employee[] employees) {
+        double sum = 0;
+        if (employees != null) {
+            for (Employee employee : employees) {
+                sum += employee.getSalary();
+            }
+        }
+        return sum;
+    }
+
+    public static double minSalary(Employee[] employees) {
+        if (employees != null) {
```

Comment:
Jeżeli w pliku nie będzie danych, to tablica nie będzie nullem, tylko raczej będzie miała length=0, uwzględnij to w warunku poprzez koniunkcję &&